person.rb

```diff
@@ -22,6 +24,10 @@ def validade_name
     @name = @corrector.correct_name(@name)
   end
 
+  def create_rental(_person, date)
```

Comment:
- [ ] To the create a `new rental` inside the `person class`, `book` and `date` should be passed in as parameters to the `create_rental` method. The person itself should be sent as a parameter to create the new rental achieving the `'has-many'` association. Please kindly fix that.

Reply:
Thanks for the review.

I fixed this here: https://github.com/leolpaz/School-Library/pull/3/commits/d25d5bd44cce2e81234c38f7be0405eb653aae05
It was fixed sometime after asking for the review, just commenting it here to let the next reviewer know why there is no commit after this review.